x-pack/docs/en/security/authentication/oidc-guide.asciidoc

```diff
@@ -369,6 +381,30 @@ will trigger re-authentication of the user. For instance, when using OpenID Conn
 single sign-on to {kib}, this could be set to +$\{kibana-url}/logged_out+,  which will show a user-
 friendly message to the user.
 
+[[oidc-ssl-config]]
+==== OpenID Connect Realm SSL Configuration
+
+OpenID Connect depends on TLS to provide security properties such as encryption in transit and endpoint authentication. The RP
+is required to establish back-channel communication with the OP in order to exchange the code for an ID Token during the
+Authorization code grant flow and in order to get additional user information from the UserInfo endpoint. Furthermore, if
+you configure `op.jwks_path` as a URL, {es} will need to get the OP's signing keys from the file hosted there. As such, it is
+important that {es} can validate and trust the server certificate that the OP uses for TLS. Since the system truststore is
+used for the client context of outgoing https connections, if your OP is using a certificate from a trusted CA, no additional
+configuration is needed.
+
+However, if your OP uses a certificate that is issued for instance, by a CA used only in your Organization, you must configure
```

Comment:
```suggestion
However, if the issuer of your OP's certificate is not trusted by the JVM on which {es} is running (e.g it uses a organization CA), then you must configure
```

I found the previous version a little hard to read, but it also implies that organisational CAs will require special configuration when it is often the case that the corporate CA has already been added to the JRE's default trust store.